### PR TITLE
Moved the commands to appropriate modules.

### PR DESCRIPTION
**Pull request by yhontyk**
Moved a few commands to appropriate modules.

Addresses these two issues:
- https://github.com/openshift/openshift-docs/issues/20136
- https://github.com/openshift/openshift-docs/issues/20134

Branches 4.2, 4.3, 4.4

### CODE REVIEW

File: modules/openshift-developer-cli-creating-and-deploying-a-nodejs-application-with-odo.adoc [hunk 1]
```diff
@@ -8,10 +8,45 @@
 
 .Procedure 
 
-. Change the current directory to the front-end directory:
+ifeval::["{context}" == "creating-a-single-component-application-with-odo"]
```
Comment (kalexand-rh) [inline modules/openshift-developer-cli-creating-and-deploying-a-nodejs-application-with-odo.adoc:11]: We've switched to using the ifeval statements to set variables based on the context statements and then just use if statements to conditionalize the text, like here: https://raw.githubusercontent.com/openshift/openshift-docs/master/modules/installation-initializing.adoc

File: modules/openshift-developer-cli-creating-and-deploying-a-nodejs-application-with-odo.adoc [hunk 2]
```diff
@@ -8,10 +8,45 @@
...
+
+. Create a new directory for your components:
```
Comment (kalexand-rh) [inline modules/openshift-developer-cli-creating-and-deploying-a-nodejs-application-with-odo.adoc:13]: ~new~

File: modules/openshift-developer-cli-creating-and-deploying-a-nodejs-application-with-odo.adoc [hunk 3]
```diff
@@ -8,10 +8,45 @@
```
Comment (kalexand-rh) [inline modules/openshift-developer-cli-creating-and-deploying-a-nodejs-application-with-odo.adoc:8]: You need a short context statement that describes the purpose of the steps. All procedure modules need to have them.

File: modules/openshift-developer-cli-deploying-the-back-end-component.adoc [hunk 4]
```diff
@@ -31,10 +31,22 @@ nodejs        openshift     10,8,8-RHOAR,latest
 openjdk18     myproject     latest
 ----
 
+. Create a new directory for your components:
```
Comment (kalexand-rh) [inline modules/openshift-developer-cli-deploying-the-back-end-component.adoc:34]: ~new~

File: modules/openshift-developer-cli-creating-a-project-in-odo.adoc [hunk 5]
```diff
@@ -15,48 +15,10 @@
...
-
-. Download the example front-end application:
-+
-----
-$ git clone https://github.com/openshift/nodejs-ex
-----
-
-endif::[]
-
 . Create a new project:
```
Comment (kalexand-rh) [inline modules/openshift-developer-cli-creating-a-project-in-odo.adoc:18]: ~new~

File: modules/openshift-developer-cli-creating-and-deploying-a-nodejs-application-with-odo.adoc [hunk 6]
```diff
@@ -8,10 +8,45 @@
...
+----
+
+. Download the example Node.js application:
++
+----
+$ git clone https://github.com/openshift/nodejs-ex
+----
+endif::[]
+
+ifeval::["{context}" == "creating-a-multicomponent-application-with-odo"]
```
Comment (kalexand-rh) [inline modules/openshift-developer-cli-creating-and-deploying-a-nodejs-application-with-odo.adoc:26]: Please update the list of assemblies that use this module.

File: modules/openshift-developer-cli-creating-and-deploying-a-nodejs-application-with-odo.adoc [hunk 7]
```diff
@@ -8,10 +8,45 @@
...
+
+. Download the example back-end and front-end applications:
++
+----
+$ git clone https://github.com/openshift-evangelists/Wild-West-Frontend frontend
```
Comment (kalexand-rh) [inline modules/openshift-developer-cli-creating-and-deploying-a-nodejs-application-with-odo.adoc:31]: Does support know that you have instructions to clone content from github?

**Review by kalexand-rh — COMMENTED** [review]
I have a couple of changes.

File: modules/openshift-developer-cli-creating-and-deploying-a-nodejs-application-with-odo.adoc (hunk 7 above)
Comment (boczkowska) [reply modules/openshift-developer-cli-creating-and-deploying-a-nodejs-application-with-odo.adoc:31]: Yes, we've approved all of our external URLs with GSS.

File: modules/openshift-developer-cli-creating-and-deploying-a-nodejs-application-with-odo.adoc (hunk 1 above)
Comment (boczkowska) [reply modules/openshift-developer-cli-creating-and-deploying-a-nodejs-application-with-odo.adoc]: Oh, I missed that change! Sorry, fixed. The build looks fine, so, hopefully, I've done in correctly. 
What was the reasoning for switching to variables?

File: modules/openshift-developer-cli-creating-and-deploying-a-nodejs-application-with-odo.adoc (hunk 3 above)
Comment (boczkowska) [reply modules/openshift-developer-cli-creating-and-deploying-a-nodejs-application-with-odo.adoc:14]: I've added introductions for modules that this PR touched. Some other modules might miss intros, I'll scope and update them separately so this PR won't blow out of proportion.

**Comment by boczkowska** [comment]
@kalexand-rh thank you for your feedback. I've addressed all of the issues you pointed out and also squashed the commits. Please take a look when you have time.

File: modules/openshift-developer-cli-creating-and-deploying-a-nodejs-application-with-odo.adoc (hunk 1 above)
Comment (kalexand-rh) [reply modules/openshift-developer-cli-creating-and-deploying-a-nodejs-application-with-odo.adoc]: You can't put two arguments in an ifeval statement, so if you need to use the same text in multiple locations, you have to duplicate it. It's a good way to make it easier to reuse content as you use it in more locations.

File: modules/openshift-developer-cli-creating-and-deploying-a-nodejs-application-with-odo.adoc (hunk 3 above)
Comment (kalexand-rh) [reply modules/openshift-developer-cli-creating-and-deploying-a-nodejs-application-with-odo.adoc:14]: I think it's a good improvement to make as you touch modules or when you get space to work on RFEs.

File: modules/openshift-developer-cli-deploying-the-front-end-component.adoc [hunk 8]
```diff
@@ -3,12 +3,27 @@
...
+ifeval::["{context}" == "creating-a-multicomponent-application-with-odo"]
+:multi:
+endif::[]
+ifeval::["{context}" == "creating-an-application-with-a-database"]
+:database:
+endif::[]
+
 [id="deploying-the-front-end-component_{context}"]
 
+To create and deploy a front-end component, download the Node.js application and push the source code to your cluster with `{odo-title}`.
```
Comment (kalexand-rh) [inline modules/openshift-developer-cli-deploying-the-front-end-component.adoc:15]: @boczkowska, please put this after the heading.

File: modules/openshift-developer-cli-creating-and-deploying-a-nodejs-application-with-odo.adoc (hunk 1 above)
Comment (boczkowska) [reply modules/openshift-developer-cli-creating-and-deploying-a-nodejs-application-with-odo.adoc]: Thank you! That makes total sense.

File: modules/openshift-developer-cli-deploying-the-front-end-component.adoc (hunk 8 above)
Comment (boczkowska) [reply modules/openshift-developer-cli-deploying-the-front-end-component.adoc]: Oops, sorry about that. Fixed.

**Pull request merged**